libmscore/scorefile.cpp

```diff
@@ -462,8 +457,11 @@ bool MasterScore::saveFile(bool generateBackup)
 //                               tr("Removing old backup file %1 failed").arg(backupName));
                         }
                   }
+            //
+            // step 2
```

Comment:
The "step 2" should probably remain in the same position.

Reply:
@Marr11317 ?

Reply:
wow, I completely forgot that one...
However, I am not sure that it really should. The step has moved, so why keep the name of it at some other place?